staging/kube-oidc-proxy/values.yaml

```diff
@@ -0,0 +1,66 @@
+# Default values for kube-oidc-proxy.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
```

Comment:
It might be worth commenting the reason here why a single replica is the default? I'm not suggesting it as a problem, but it occurred to me that right here could be a good place to add some context about the operational concerns of the app.